integration_test/support/schemas.exs

```diff
@@ -271,3 +271,50 @@ defmodule Ecto.Integration.PostUserCompositePk do
     timestamps()
   end
 end
+
+defmodule Ecto.Integration.Hotel do
+  @moduledoc """
+  This module is used to test:
+
+    * Ecto.assoc schema_prefix on through-associations
+
+  """
+  use Ecto.Integration.Schema
```

Comment:
I would like to avoid adding multiple schemas more. Could we do a single new schema? Maybe call it PostWithPrefix?